phpseclib/Crypt/RSA.php

```diff
@@ -250,12 +250,20 @@ class RSA
     /**#@-*/
 
     /**
+     * Singleton instance
```

Comment:
It's a holdover from #815. Initially an RSA instance was being used to return a string but I made it return objects instead and I guess I forgot to remove this bit of the code.